contracts/solidity/.env.example

```diff
@@ -1,3 +1,10 @@
-GOERLI_RPC_URL=
-PRIVATE_KEY=
-ETHERSCAN_API_KEY=
+# To deploy Ethereum contract 
+ETHERSCAN_API_KEY=<ETHERSCAN_API_KEY>
+ETH_PRIVATE_KEY=<ETH_PRIVATE_KEY>
+GOERLI_RPC_URL=<GOERLI_RPC_URL>
+
+SN_MESSAGING_ADDRESS=0xde29d060D45901Fb19ED6C6e959EB22d8626708e
+SN_WITHDRAW_SELECTOR=0x15511cc3694f64379908437d6d64458dc76d02482052bfb8a5b33a72c054c77 
```

Comment:
same here

Reply:
good catch this shouldn't be hardcoded at all we have to be careful with this

Reply:
we need to double check how this address works so we don't make a mistake here

Reply:
This has been resolved in a pull request that has already been merged into the main branch (this branch is already aligned with that).